### DISCUSSION TITLE
How exactly do we run this?

### DISCUSSION THREAD

**Discussion by wiebenor** [diStyApps/Safe-and-Stable-Ckpt2Safetensors-Conversion-Tool-GUI#12] [Q&A] [answered]
I'm running windows, and I have miniconda installed, and also plan on installing python eventually if it's ever needed, and I'm wondering if this has instructions somewhere to run the app and or install any dependencies required??? In what little small amount of reading I did on the main page I didn't see anything about installing it but then again I don't know maybe I missed something?

**Comment by wiebenor** [comment]
Oops... Think I downloaded the wrong thing... Downloading the right thing might help me

**Comment by diStyApps** [comment, marked as answer]
Thanks for using the app!

I see you figured out how to start the app, Here are the quick instructions for other users until I upload a proper guide. 

To start the app on a Windows machine, follow these steps:

Go to the [releases](https://github.com/diStyApps/Safe-and-Stable-Ckpt2Safetensors-Conversion-Tool-GUI/releases) page 
Download the latest release of "Safe-and-Stable-Ckpt2Safetensors-GUI"
Unzip the file
Navigate to the folder where the file was extracted
Click on "run_app_gui.exe" to launch the app.

**Reply by Emmadgithub** [reply]
> Thanks for using the app!
> 
> I see you figured out how to start the app, Here are the quick instructions for other users until I upload a proper guide.
> 
> To start the app on a Windows machine, follow these steps:
> 
> Go to the [releases](https://github.com/diStyApps/Safe-and-Stable-Ckpt2Safetensors-Conversion-Tool-GUI/releases) page Download the latest release of "Safe-and-Stable-Ckpt2Safetensors-GUI" Unzip the file Navigate to the folder where the file was extracted Click on "run_app_gui.exe" to launch the app.

I have no idea to run this in M1 MAC ventura, I have Annaconda and Python too
